ydb/core/cms/console/configs_dispatcher.cpp

```diff
@@ -233,6 +234,7 @@ class TConfigsDispatcher : public TActorBootstrapped<TConfigsDispatcher> {
             IgnoreFunc(TEvConsole::TEvGetNodeConfigResponse);
             // Pretend we got this
             hFuncTraced(TEvConsole::TEvConfigNotificationRequest, Handle);
+            hFuncTraced(TEvConsole::TEvGetNodeConfigurationVersionRequest, Handle);
```

Comment:
It will kill (abort) node if there won't be any protection from usage with console on new version. We need at least feature_flag to keep clusters safe.